pkg/app/piped/appconfigreporter/appconfigreporter.go

```diff
@@ -383,6 +383,10 @@ func (r *Reporter) readApplicationInfo(repoDir, repoID, cfgRelPath string) (*mod
 	if spec.Name == "" {
 		return nil, fmt.Errorf("missing application name: %w", errMissingRequiredField)
 	}
+	envName := ""
+	if e, ok := spec.Labels[config.EnvLabelKey]; ok {
+		envName = e
+	}
```

Comment:
I'm sorry if I make a mistake.
Is it better that do not make breaking changes?
```suggestion
	envName := spec.EnvName
	if e, ok := spec.Labels[config.EnvLabelKey]; ok {
		envName = e
	}
```

Reply:
@knanao is correct. I'm afraid we should keep both to avoid a breaking change.

Reply:
For sure!

Reply:
This was released in the previous version, so users suppose to be fewer. Let's remove this after the next version.